graylog2-server/src/main/java/org/graylog/security/authservice/AuthServiceBackendConfig.java

```diff
@@ -40,7 +41,7 @@ default void validate(ValidationResult result) {
     }
 
     @JsonIgnore
-    List<String> hostAllowList();
+    Optional<List<String>> externalHTTPHosts();
```

Comment:
Let's add some javadoc here to help developers to know what this is used for. We can mention CSP as an example use case.

Reply:
done